libs/ojsonnet/Eval_jsonnet.ml

```diff
@@ -31,18 +30,19 @@ let logger = Logging.get_logger [ __MODULE__ ]
 (*****************************************************************************)
 (* Types and constants *)
 (*****************************************************************************)
-type env = {
-  (* The spec uses a lambda-calculus inspired substitution model, but
-   * it is probably simpler and more efficient to use a classic
-   * environment where the locals are defined. Jsonnet uses lazy
-   * evaluation so we model this by using Lazy below.
-   *)
-  locals : (local_id, V.value_ Lazy.t) Map_.t;
-  (* for call tracing *)
-  depth : int;
-}
 
-and local_id = LSelf | LSuper | LId of string
+(*type env = {
```

Comment:
This comment should be removed (and the environment type moved to Value_jsonnet.ml instead).